lib/static_data_repo.dart

```diff
@@ -0,0 +1,90 @@
+class StaticDataRepo {
```

Comment:
would it be easier to manage this as just html files inside assets?

Reply:
Yep! The're text fragments rather than full html pages, but it might be easier. Will investigate. Would be good to discuss how to reduce long-term data maintenance, making the UI more data-driven etc. I am concerned about how fragile this (ie my own) structure is

Reply:
refactored to html files with a FutureBuilder loader